schemas_extra.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "fileMatch": ["*.sublime-build"],
+    "url": "sublime://schemas/sublime-build"
+  },
+  {
+    "fileMatch": ["*.sublime-color-scheme", "*.hidden-color-scheme"],
+    "url": "sublime://schemas/sublime-color-scheme"
+  },
+  {
+    "fileMatch": ["*.sublime-commands"],
+    "url": "sublime://schemas/sublime-commands"
+  },
+  {
+    "fileMatch": ["*.sublime-keymap"],
+    "url": "sublime://schemas/sublime-keymap"
+  },
+  {
+    "fileMatch": ["*.sublime-macro"],
+    "url": "sublime://schemas/sublime-macro"
+  },
+  {
+    "fileMatch": ["*.sublime-menu"],
+    "url": "sublime://schemas/sublime-menu"
+  },
+  {
+    "fileMatch": ["*.sublime-mousemap"],
+    "url": "sublime://schemas/sublime-mousemap"
+  },
+  {
+    "fileMatch": ["*.sublime-project"],
+    "url": "sublime://schemas/sublime-project"
+  },
+  {
+    "fileMatch": ["*.sublime-settings"],
+    "url": "sublime://schemas/sublime-settings"
+  },
+  {
+    "fileMatch": ["*.sublime-theme", "*.hidden-theme"],
+    "url": "sublime://schemas/sublime-theme"
+  },
+  {
+    "fileMatch": ["/.vscode/*.json"],
+    "url": "sublime://schemas/vscode-base"
+  },
+  {
+    "fileMatch": ["/package.json"],
+    "url": "sublime://schemas/package-json-eslint"
+  },
+  {
+    "fileMatch": ["tsconfig.json", "tsconfig.*.json", "tsconfig-*.json"],
+    "url": "sublime://schemas/tsconfig"
```

Comment:
I think that you don't need "sublime://schemas/tsconfig" and "sublime://schemas/jsconfig" because they are already defined in `schemas.json` file :)

Reply:
The ones that are in schemas.json are official specs that abide schema drafts while those here are additional augmentations for those official ones that contain noon-standard properties (vscode extensions). To allow comments, trailing commas and a couple other things you have to use those non-standard properties.
It's ok to provide more than one schema for same file pattern